src/static/javascripts/components/questions.jsx

```diff
@@ -82,7 +82,7 @@ var MultipleChoice = React.createClass({
 * Single Choice
 */
 var SingleChoice = React.createClass({
```

Comment:
Let's meet and compare our card functions just to make sure we're on the same page. I had to change  the handleChange function and I just want to make sure we're not messing something up.